documentation/specs/FocusedProductLayout.md

```diff
@@ -0,0 +1,215 @@
+# `FocusedProductLayout` Component Specification
```

Comment:
not entirely sure yet if this will end up being a variant of `ProductLayout` as defined previously, or a separate component like it's defined here. the API surface are has some shared properties but it also has some significant differences. i'm going to talk to design about if there are any plans for transitioning between layouts, and i think that could affect the decision path here. i anticipate fleshing this out in implementation